hescorehpxml/base.py

```diff
@@ -2271,6 +2288,8 @@ def get_generation(self, b):
                 old_div(sum([year * capacity for year, capacity in zip(years, capacities)]), total_capacity))
             wtavg_azimuth = old_div(sum(
                 [az * capacity for az, capacity in zip(azimuths, capacities)]), total_capacity)
+            wtavg_tilt = old_div(sum(
+                [t * capacity for t, capacity in zip(tilts, capacities)]), total_capacity)
```

Comment:
We're moving away from `old_div` which was a relic of the python2to3 conversion. @bpark1327 has fixed all the rest of them in #160. 

```suggestion
            wtavg_tilt = sum(t * capacity for t, capacity in zip(tilts, capacities)) / total_capacity
```